modules/firemission/functions/FIREMIS/fn_FIREMIS_DoConstantArtillery.sqf

```diff
@@ -0,0 +1,42 @@
+// Author: StatusRed
+#include "script_component.hpp"
+
+/**
+ * FW_fnc_FIREMIS_DoConstantArtillery
+ * Commands a battery to constantly fire artillery at a marker target with unliminted ammo until the battery is destroyed.
+ * Author: StatusRed
+ *
+ * Parameters:
+ * 0: unit - The unit to command fire
+ * 1: target - The target to fire at
+ * 2: dispersion - How close to the target to fire within
+ * 3: minDelay - The minimum delay between firing (in seconds)
+ * 4: delayModifier - A range from 0 to the delay modifier value (in seconds) is added to the minimum delay
+*/
+params[
+  ["_unit", objNull, [objNull]],
+  ["_target", objNull, ["", objNull, []]],
+  ["_dispersion", 100, [100]],
+  ["_minDelay", 60, [60]],
+  ["_delayModifier", 61, [61]]
+];
+
+if (isServer) then {
```

Comment:
so this is all being done on server, there is no HC execution of the module for functions like this?

Reply:
> 

I've mainly put this check there to ensure that the code is only run once. For example, if someone sticks the call in the settings.sqf (which is loaded by clients and the server) it might end up getting run multiple times.

Reply:
gotcha